docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -462,6 +511,35 @@ After training, the model must be destroyed with `ml_train_model_destroy()`.
 `ml_train_model_add_layer()`, `ml_train_set_optimizer()`, and `ml_train_set_dataset()` transfers ownership to the model.
 `layers`, `optimizers` and `dataset` that belongs to the `model` are also deleted.
 
+## Using the trained model for the inference
```

Comment:
How about this heading? 

## Use trained model for inference

Reply:
Changed as suggested